### DISCUSSION TITLE
Any ideas on how to trigger a glb animation from within Spoke?

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2844] [General Questions & Discussion] [answered]
At present, we can only play and loop on load, no other controls. Would be really useful.

**Comment by johnshaughnessy** [comment, marked as answer]
I agree it would be really useful. The feature request is being tracked here : https://github.com/mozilla/hubs/issues/2635

**Comment by ocularvr** [comment]
I have a feeling that we can use the chat window to send arguments like start/stop animation on a glb. Who has the key to the chat black box? 
:)

**Reply by emclaren** [reply]
You are welcome to add comments to #2635 if this is a feature you would like to see included 😀
